src/lib/db/migrate.ts

```diff
@@ -21,6 +24,12 @@ export const migrate = async (prod: boolean, log: Logger): Promise<void> => {
 		return;
 	}
 
+	if (prod) {
+		log.info('backing up db');
+		await spawn('sudo', ['-i', '-u', 'postgres', 'pg_dump', 'felt', '-f', `${BACKUP_FILE}`]);
+		await spawn('mv', [`${BACKUP_DIR}/${BACKUP_FILE}`, '.']);
```

Comment:
```suggestion
		await spawn('sudo', ['mv', `${BACKUP_DIR}/${BACKUP_FILE}`, '.']);
```

fails for me locally without this